src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
```

Comment:
Don't keep all logic in a single package. Consider moving your operation handler classes into a separate package, such as `strategy`, to better organize your code and adhere to the checklist.